flink-table/flink-table-planner-blink/src/main/java/org/apache/calcite/avatica/util/DateTimeUtils.java

```diff
@@ -309,6 +309,28 @@ public static SimpleDateFormat newDateFormat(String format) {
 		return sdf;
 	}
 
+	/** Helper for CAST({timestamp} Or {date} or {time} AS VARCHAR(n)). */
+	public static String unixDateTimeToString(Object o, TimeZone tz) {
```

Comment:
This method is only used in `TestSinkUtil`, can we move it to `TestSinkUtil`? 

We want to avoid to maintain files copied from Calcite.

Reply:
Sure, I will move the method to `TestSinkUtil`.
But almost all method in `DateTimeUtils` are copied from Calcite……